code/style.css

```diff
@@ -1,7 +1,318 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
-
-.active + .answer {
-  /* some styling here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  font-family: "Heebo", sans-serif;
+  background-color: #dedcdc;
+}
+.video-header {
+  height: 95vh;
+  width: 100vw;
+  position: relative;
+  display: flex;
+  overflow: hidden;
+}
+.video-header video {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  object-fit: cover;
+  top: 0;
+  left: 0;
+}
+.logo {
+  color: white;
+  font-size: 60px;
+  font-family: "Abril Fatface";
+  position: absolute;
+  z-index: 1;
+  padding: 10px 30px;
+}
+.tagline {
+  width: 700px;
+  position: absolute;
+  align-self: center;
+  right: 10%;
+  color: white;
+}
+.tagline h3 {
+  font-family: "Merriweather", serif;
+  font-style: italic;
+  font-size: 43px;
+  margin-bottom: 10px;
+}
+.tagline p {
+  margin-bottom: 10px;
+  font-size: 18px;
+}
+.tagline button {
+  padding: 0.5em 3em;
+  border: 0.16em solid #ffffff;
+  margin: 0 0.3em 0.3em 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #ffffff;
+  text-align: center;
+  transition: all 0.15s;
+  background-color: rgba(96, 87, 155, 0.205);
+  border-radius: 15px;
+}
+.tagline button:hover {
+  color: #c1b3cc;
+  border-color: #9f94a8;
+}
+/* Here I wanted to do a JS button transition to the Newsletter section but I didn't manage to solve it this week */
+.tagline button:active {
+  color: #c1b3cc;
+  border-color: #9f94a8;
+}
+.main {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+.faq,
+.newsletter {
+  background-color: white;
+  width: 80%;
+  max-height: 80%;
+  margin: auto;
+  padding: 20px;
+  overflow: scroll;
+  box-shadow: 2px 2px 4px #525f4d;
+}
+.faq h2 {
+  font-family: "Merriweather", serif;
+  font-style: italic;
+  font-size: 30px;
+  color: #3e3f4b;
+  padding-bottom: 10px;
+}
+/* Accordion styling */
+button.accordion {
+  width: 100%;
+  border: none;
+  outline: none;
+  text-align: left;
+  padding: 15px 20px;
+  font-size: 16px;
+  color: #333;
+  transition: background-color 0.2s linear;
+  margin-bottom: 3px;
+}
+button.accordion:after {
+  content: "+";
+  font-size: 18px;
+  float: right;
+}
+button.accordion.is-open:after {
+  content: "-";
+}
+button.accordion:hover,
+button.accordion.is-open {
+  background-color: rgb(217, 211, 224);
+}
+.accordion-content {
+  background-color: rgb(233, 229, 240);
+  font-size: 14px;
+  color: #3e3f4b;
+  width: 100%;
+  padding: 0 20px;
+  margin-bottom: 10px;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-in-out;
+}
+/* Accompanying image */
```

Comment:
Nice with a comment to what this is regarding!

Reply:
Your faq-image, as well as the newsletter image further down, looks really nice and fit well together with the purpose of your page. Opacity is perfect and they do fit really well next to the faq accordion + newsletter.